EurorackPanelDesigner/EurorackPanelDesigner.inx

```diff
@@ -8,7 +8,8 @@
   <param name="desc_unit" type="description">(Eurorack panels are usually 3U 128.5mm high)</param> 
   <param name="height" type="optiongroup" appearance = "combo" gui-text="Panel height">
     <option value="128.5">3U</option>
-	<option value="36.95">1U</option>
```

Comment:
Pretty sure the 6 and 9 got switched somehow, this now follows the [spec](https://intellijel.com/support/1u-technical-specifications/).